interactive list implementations

Wikitelmex opened the pull request:
- add js file and imported inside main.js
- add methods to the new file to update the status of the tasks
- add checkboxes events to save completed status
- save status in the current array and in local storage

wikitelmex1 committed:
interactive list implementations

- add js file and imported inside main.js
- add methods to the new file to update the status of the tasks
- add checkboxes events to save completed status
- save status in the current array and in local storage

barackm reviewed (approved):
Hi @Wikitelmex, 👋 

# STATUS: Approved  🎉 
Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 

![](https://media.giphy.com/media/vmtxnxveVUodG/giphy.gif)

- [ ] [OPTIONAL] It could be a good idea to keep committing often, because for now, I can see that you only have a single commit, having multiple commits helps us come back to a previous state of the project easily in case of a problem.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

wikitelmex1 committed:
long break commit

wikitelmex1 committed:
crud actions

- add new task
- edit existing task
- delete existing task
- delete all completed tasks
- save in localstorage all the changes made in tasks

wikitelmex1 committed:
corrections in live demo

- update index.html
- update main.js
